test/outdated/outdated_test.dart

```diff
@@ -380,6 +380,59 @@ Future<void> main() async {
     await ctx.runOutdatedTests();
   });
 
+  testWithGolden('do not report ignored advisories', (ctx) async {
+    final builder = await servePackages();
+    builder
+      ..serve('foo', '1.0.0', deps: {'transitive': '^1.0.0'})
+      ..serve('transitive', '1.2.3');
+
+    await d.dir(appPath, [
+      d.pubspec({
+        'name': 'app',
+        'dependencies': {
+          'foo': '^1.0.0',
+        },
+        'ignored_advisories': ['ABCD-1234-5678-9101'],
+      }),
+    ]).create();
+    await pubGet();
+
+    builder.affectVersionsByAdvisory(
```

Comment:
Could you make the test such that it has one advisory that's ignored and one that's not. (And one that's ignored via an alias?

Reply:
I would like to keep the vanilla tests, so that we can cover the boolean in the json output. I'll add extra tests for the combined cases.

Reply:
Ah - that makes sense - but you could let the ignored one be the one that is affecting the current version?

Reply:
Yes, I think an extra test is also fine.

Reply:
It's ok if you prefer an extra test - just know that these tests are relatively expensive (because they run `pub outdated` in quite a number of different configs.

Reply:
In general I prefer two simple tests over one covering too much. I can combine the two vanilla tests, though.

Reply:
SG